test/e2e/e2e_test.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2025.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog/v2"
+
+	v1 "github.com/openshift/api/operator/v1"
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
+)
+
+var _ = Describe("LWS Operator", Ordered, func() {
+	It("Verifying that conditions are correct", func() {
+		ctx := context.TODO()
+		lwsOperators, err := clients.LWSOperatorClient.List(ctx, metav1.ListOptions{})
+		Expect(err).NotTo(HaveOccurred())
+
+		Expect(len(lwsOperators.Items)).To(Equal(1))
+
+		By("checking no degraded condition exists")
+		degraded := false
+		for _, lwsOperator := range lwsOperators.Items {
+			for _, condition := range lwsOperator.Status.Conditions {
+				if strings.HasSuffix(condition.Type, v1.OperatorStatusTypeDegraded) {
+					if condition.Status == v1.ConditionTrue {
+						degraded = true
+					}
+				}
+			}
+		}
+		Expect(degraded).NotTo(BeTrue())
```

Comment:
when this happens, it would be useful to output the full condition, e.g. name/reason